### PR TITLE
Add possibility for themes to change text color in header

### LINKED ISSUE #12201: Customizable LESS var for header/menu links color
Hi all,

I already dicussed on forum.piwik.org about that: https://forum.piwik.org/t/customize-header-menu-links-color-within-a-theme/23418

I wish to have the oportunately to define the 
 * header-background color
 * header-menu color 
within a theme.

But there is only the following:
`@theme-color-header-background: #ffffff;`

It would be nice to have also e.g. a
`@theme-color-menu-text: #ffffff;`

Current-version: 3.2.0

Thx

### PR DESCRIPTION
**Pull request by tsteur**
fixes #12201 If I understand it correctly, the color in the header should be changeable which I totally agree to in order to make it useful.

Expecting UI tests to fail for the theming part with black menu text color. 

For the variable I used the ending `-text` even though they are technically links (`-link`) to be consistent with the menu.

### CODE REVIEW

**Comment by tsteur** [comment]
UI test failures are not from this PR

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ This is the Developer Changelog for Piwik platform developers. All changes in ou
 
 The Product Changelog at **[piwik.org/changelog](https://piwik.org/changelog)** lets you see more details about any Piwik release, such as the list of new guides and FAQs, security fixes, and links to all closed issues. 
 
+## Piwik 3.2.1
+
+### New APIs
+ * Themes can now customer the header text color using `@theme-color-header-text`
```
Comment (sgiehl) [inline CHANGELOG.md]: guess that should be `customize`

**Review by sgiehl — CHANGES_REQUESTED** [review]
Left a comment. Besides that, looks good

**Review by sgiehl — APPROVED** [review]
Fixed the typo. Now LGTM

**Pull request merged**